packages/flutter_tools/.gitignore

```diff
@@ -0,0 +1 @@
+lib/cache/
```

Comment:
We won't actually need this once my patch lands.  I'm moving the cache up to `//bin/cache` to share with everything else.